src/App.tsx

```diff
@@ -31,65 +32,127 @@ export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (!SortType.NONE) {
```

Comment:
The condition `!SortType.NONE` will always be true, as `SortType.NONE` is 0 and `!0` is true. Instead of checking for `!SortType.NONE`, check for `sortType !== SortType.NONE`.